pkg/Directory.Build.props

```diff
@@ -24,6 +27,17 @@
     <PackageReleaseNotes>https://github.com/dotnet/machinelearning/tree/master/Documentation/release-notes</PackageReleaseNotes>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Content Include="$(PackageAssetsPath)$(PackageIdFolderName)\lib\**\*" Pack="true" PackagePath="lib" />
+    <Content Include="$(PackageAssetsPath)$(PackageIdFolderName)\runtimes\**\*" Pack="true" PackagePath="runtimes" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(IsSymbolsPackage)' != 'true'">
```

Comment:
there will be some symbols with the extension .dll, this wont remove them

Reply:
what symbols have an extension of `.dll`?

Reply:
Sorry, my bad I got confused.